### ISSUE TITLE
Toggle VIdeo Settings

### ISSUE THREAD

**Issue by trymeouteh** [mirdaki/ping#17]
Please have a setting in the client to enable/disable video in all calls.

**Comment by mirdaki** [comment]
To clarify, what your asking for is to have a setting to always have it enabled or disabled?

Would having to explicitly enable video for each call (like clicking a camera button) meet what you're asking for?

**Comment by trymeouteh** [comment]
> To clarify, what your asking for is to have a setting to always have it enabled or disabled?

Yeah a setting. Since Ping is going to be a gaming client, gamers dont use video chat when gaming with their friends, they only use their mic and text chat. Therefore if the client is going to support video which I don't think Ping needs to support video, it should be disabled by default and need to be enabled in the settings.

> Would having to explicitly enable video for each call (like clicking a camera button) meet what you're asking for?

That can work but please make a setting on weather this video camera button is present in the client or not to prevent gamers from accidentally clicking on it.

I would prefer a setting in the client to enable/disable video chat and if the video chat is disabled to not have a camera button in the audio chat to toggle your camera on/off.

And I think Ping should not grab video from others in the chat even if another user is using video chat on Element for example since Ping should be a low bandwidth, fast gaming chat client and video takes up bandwidth unlike audio calls.

**Comment by mirdaki** [comment]
Thanks for the input! I fully agree voice chat should be a priority over video chat and both should have some user control over bandwidth. The UI design is still super rough, but I'll keep in mind this as it goes.
